### PR TITLE
Fix #1175 query events

### LINKED ISSUE #1175: Slice dash interaction with variant level export
- [x] variant export broken for (at least) cust002: using sanger_ordered as a proxy for validation is not enough as variants can be marked true / false at the pin level without a sanger_order.
- [ ] totals on the variants is for a slice. Actually use a sliced query anyway for the export? That is best given the page as it is today, otherwise let's make a more clear boundary btw the sliced and unsliced part.

Sorry, I didn't pay enough attention to this when the two branches merged - and the tests were presumably not geared towards finding this issue.

### PR DESCRIPTION
**Pull request by northwestwitch**
1. Collect validated and validable variants using event collection
2. Splitted the dashboard into 3 tabs
3. Variant tab is showing panels even if there are no validated variants yet

### CODE REVIEW

**Comment by northwestwitch** [comment]
This branch is on stage now

File: scout/server/blueprints/dashboard/controllers.py [hunk 1]
```diff
@@ -23,130 +23,143 @@ def get_dashboard_info(adapter, institute_id=None, slice_query=None):
...
+        }
+    )
+
+    # taking into account that var_valid_orders might be 0:
+    percent_validable = 0
+    percent_validated = 0
+
+    if var_valid_orders:
+        percent_validable = len(validated_tp) / var_valid_orders
+        percent_validated = len(validated_fp) / var_valid_orders
```
Comment (dnil) [inline scout/server/blueprints/dashboard/controllers.py]: Could we change the name of these? percent_validated_tp, percent_validated_fp perhaps?

File: scout/server/blueprints/dashboard/controllers.py [hunk 2]
```diff
@@ -23,130 +23,143 @@ def get_dashboard_info(adapter, institute_id=None, slice_query=None):
...
+    # append
+    overview.append(
+    {
+        'title': 'Validation ordered',
+        'count': n_validation_cases,
+        'percent': n_validation_cases / total_sliced_cases,
+    })
+    overview.append(
+    {
+        'title': 'Validated cases',
```
Comment (dnil) [inline scout/server/blueprints/dashboard/controllers.py]: Hm, might be misunderstood as nr of true positives. How about 'Validated cases (TP + FP)'?

File: scout/server/blueprints/dashboard/controllers.py (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/dashboard/controllers.py]: Sure, didn't realize how bad I named them

**Comment by northwestwitch** [comment]
Thanks, I'll merge it!

**Pull request merged**